frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -531,6 +540,155 @@ describe("QueryBuilder", () => {
     });
   });
 
+  describe("unsaved changes warning", () => {
+    describe("native queries", () => {
+      afterEach(() => {
+        jest.restoreAllMocks();
+      });
```

Comment:
This is safe to be removed, the reason this exists for other `describe`s is because those suites use `mockEventListener` returned from the `setup` function. And it needs to be reset manually.

```suggestion
```

Reply:
b25d16ddb6e0c989444fc6b8e868d8f79268c6c4